contracts/src/mocks/plugin/PluginMockData.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.8;
+
+import {PermissionLib} from "../../permission/PermissionLib.sol";
+
+address constant NO_CONDITION = address(0);
+
+error ConflictingValues();
+
+function mockPermissions(
+    uint160 start,
+    uint160 end,
+    PermissionLib.Operation op
+) pure returns (PermissionLib.MultiTargetPermission[] memory permissions) {
```

Comment:
Perhaps a comment is needed here explaining what this mock is for

Reply:
Added in https://github.com/aragon/osx-commons/pull/36/commits/450e00aad95552528d60280f88744b0854f8d767